packages/eslint-config-4catalyzer/rules.js

```diff
@@ -32,6 +32,20 @@ module.exports = {
     ignoreRestSiblings: false,
     argsIgnorePattern: '^_',
   }],
+  'prefer-destructuring': [
+    'error',
+    {
+      VariableDeclarator: {
+        array: false,
+        object: true,
+      },
+      AssignmentExpression: {
+        array: true,
```

Comment:
make this `false` as well? why does ESLint default this to true...

Reply:
idk it's so bad.